DPG/Tasks/AOTTrack/qaMatchEff.cxx

```diff
@@ -187,6 +187,12 @@ struct qaMatchEff {
   //      ******     BE VERY CAREFUL!   --  FILTERS !!!  *****
   //
   Filter zPrimVtxLim = nabs(aod::collision::posZ) < zPrimVtxMax;
+  Filter trackFilter = (filterbitTrackSelections.node() == 0) ||
+                       ((filterbitTrackSelections.node() == 1) && requireGlobalTrackInFilter()) || /// filterbit 4 track selections + tight DCA cuts
+                       ((filterbitTrackSelections.node() == 2) && requireGlobalTrackWoPtEtaInFilter()) ||
+                       ((filterbitTrackSelections.node() == 3) && requireGlobalTrackWoDCAInFilter()) ||
+                       ((filterbitTrackSelections.node() == 4) && requireQualityTracksInFilter()) ||
+                       ((filterbitTrackSelections.node() == 5) && requireTrackCutInFilter(TrackSelectionFlags::kInAcceptanceTracks));
```

Comment:
This filter was removed on purpose because it was giving uncorrect results. Indeed, as it was written before the PR https://github.com/AliceO2Group/O2Physics/commit/8470ceb10543e017f034381f87b1f33e8552fee0, it was cutting away all TPC-only tracks also for the denominator and the matching efficiency was always artificially at 1. Instead, the solution was to set the cuts manually as in the `track-selection` workflow and apply them with the functions `isTrackSelectedTPCCuts` and `isTrackSelectedITSCuts`